tests/test_j1939.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import mock  # type: ignore
+
+from nixnet import _cfuncs
+from nixnet import _ctypedefs
+
+from nixnet._session import j1939
+
+
+MockXnetLibrary = mock.create_autospec(_cfuncs.XnetLibrary, spec_set=True, instance=True)
+MockXnetLibrary.nx_get_property.return_value = _ctypedefs.u32(0)
+MockXnetLibrary.nx_set_property.return_value = _ctypedefs.u32(0)
+
+
+@mock.patch('nixnet._cfuncs.lib', MockXnetLibrary)
+def test_j1939_properties():
+    # This test should be expanded to cover more J1939 properties.
+    j1939.J1939.include_dest_addr_in_pgn = j1939.J1939.include_dest_addr_in_pgn
```

Comment:
My main concern with this PR is the testing strategy.

This calls the getter and setter on the upper layers but ends in a dead end at the `_cfuncs.XnetLibrary` layer.  What value does this offer us over mypy's checks?

Reply:
For methods/properties which are simply thinly layered wrappers to a library call, I've questioned the value of using unit tests over integration tests. I've advocated that we test them all in integration tests since you get the benefit of making sure that (a) the driver can correctly handle the arguments from python and that (b) python can correctly handle an actual return value from the driver. I was given the guidance that we should do some percentage of properties/methods (was it 10 or 15%, I can't remember) as integration tests and then use unit tests for the others. I have attempted to follow that guidance. It would be nice going forward if we had a testing strategy that addresses these issues.